ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/clientlibs/site/js/tabs.js

```diff
@@ -130,6 +130,37 @@
             return this.element.querySelector(Tabs.selectors.qm);
         }
 
+        #syncTabLabels(){
+            var tabs = this._elements["tab"];
+            if (tabs) {
+                tabs.forEach((tab)=>{
+                    var splitIdArr=tab.id.split("-");
+                    var replaceableId=splitIdArr[0]+"-"+splitIdArr[1];
+                    tab.id=tab.id.replace(replaceableId,this.getId());
+                    tab.setAttribute("aria-controls",
+                        tab.getAttribute("aria-controls").replace(replaceableId,this.getId()));
+                });
+            }
+        }
+
+        #syncTabPanels(){
+            var tabPanels = this._elements["tabpanel"];
+            if(tabPanels) {
+                tabPanels.forEach((tabPanel)=>{
+                    var splitIdArr=tabPanel.id.split("-");
```

Comment:
A general code formatting tip: pls try to leave spaces before and after the operator :) 

```
tabPanels.forEach((tabPanel) => {
                    var splitIdArr = tabPanel.id.split("-");
```

Reply:
Thanks , rectifying this.